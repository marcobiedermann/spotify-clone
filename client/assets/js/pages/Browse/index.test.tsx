import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import BrowsePage from '.';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <BrowsePage />
      </MemoryRouter>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
