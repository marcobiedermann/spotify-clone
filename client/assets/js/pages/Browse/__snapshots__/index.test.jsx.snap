// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <ul>
    <li>
      <a
        href="/categories"
        onClick={[Function]}
      >
        Genres & Moods
      </a>
    </li>
    <li>
      <a
        href="/featured-playlists"
        onClick={[Function]}
      >
        Featured Playlists
      </a>
    </li>
    <li>
      <a
        href="/new-releases"
        onClick={[Function]}
      >
        New Releases
      </a>
    </li>
  </ul>
</div>
`;
